packages/discv5/src/enr/setV4Crypto.ts

```diff
@@ -1,6 +1,7 @@
-// This module has the side effect of setting the ENR crypto implementations to use bcrypto
+// This module has the side effect of setting the ENR crypto implementations to use the defaultCrypto
+// implementation at present.
+// TODO: Adjust this if we use a different implementation
 
-import { setV4Crypto } from "@chainsafe/enr";
-import * as bcryptoV4Crypto from "./bcryptoV4Crypto.js";
+import { setV4Crypto, defaultCrypto } from "@chainsafe/enr";
 
-setV4Crypto(bcryptoV4Crypto);
+setV4Crypto(defaultCrypto);
```

Comment:
You can get rid of this file since we no longer need to switch implementations (and remove reference in src/index.tx)

Reply:
Do you want me to implement a wasm-crypto version of this so we have access to it in discv5?  We use [@polkadot/wasm-crypto](https://github.com/polkadot-js/wasm/tree/master/packages/wasm-crypto) in the ethereumjs-monorepo as a drop-in replacement for JS native cryptography in our client and it's something like 10x faster on average.  I haven't benchmarked it against bcrypto but it's maintained and has nice typescript definitions packaged with it.

Reply:
Okay, maybe not 10x, but here's two key functions with benchmarks by vitest for `bcrypto`, `wasm-crypto`, and `noble` 
![image](https://github.com/user-attachments/assets/7042a71b-9dcb-4787-a5cd-133ecf71083d)

I guess `bcrypto` is still the fastest since it's native c code

Reply:
I chatted with @scottypoi and we decided against adding in any extra cryptography libraries and just going with `noble`.  The main place we get a win is switching the AES stuff to nodejs builtins and then dropping `bcrypto` entirely.  

So, have gone ahead and removed the file above as suggested.